### ISSUE TITLE
Removed "self." on redundantSelf with init-only option.

### ISSUE THREAD

**Issue by r-plus** [nicklockwood/SwiftFormat#435]
```
class A {
    let orderId: String
    init(param: Parameter) {
        let detail = param.detail
        orderId = detail.orderId
    }
}
```

This code will not modified by run `swiftformat test.swift --dryrun --rules redundantSelf --self init-only --swiftversion 5.0` command. results are 

```
Running SwiftFormat...
(dryrun mode - no files will be changed.)
SwiftFormat completed in 0.07s.
0/1 files would have been formatted.
```

I'm thinking that code should insert `self.` to `orderId` property initialization line.

## Version

SwiftFormat v0.40.8

**Comment by nicklockwood** [comment]
@r-plus you're right, it should. I'll look into it.

**Comment by nicklockwood** [comment]
Fixed in version 0.40.9.
